contracts/CoinbaseResolver.sol

```diff
@@ -206,7 +207,8 @@ contract CoinbaseResolver is ERC165, Manageable, IExtendedResolver {
     }
 
     function _addSigners(address[] memory signersToAdd) private {
-        for (uint256 i = 0; i < signersToAdd.length; i++) {
+        uint256 length = signersToAdd.length;
+        for (uint256 i = 0; i < length; i++) {
```

Comment:
Technically you can save more gas here by doing `unchecked` and `++i` but I don't think it's worth hurting readability of the code to save tiny amounts of gas for a function that is rarely ever called